pkg/sessionctx/stmtctx/stmtctx.go

```diff
@@ -425,26 +429,36 @@ type StatementContext struct {
 
 // NewStmtCtx creates a new statement context
 func NewStmtCtx() *StatementContext {
```

Comment:
Not related with this PR. Is it possible to use `NewStmtCtxWithTimeZone(time.UTC)` to implement `NewStmtCtx`?

Reply:
Yes, I think it is a better way when we have more fields to init